<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QMainWindow" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>morse - Qt Editon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tbDecoder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbEncoder">
     <attribute name="title">
      <string>Encoder</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Insert Plain Text:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtPlainEncode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Encoded Value:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtEncEncode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>431</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Decoder</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Insert Morse Code:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtDecDecode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Decoded Value:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtPlainDecode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>431</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuFrom_File">
     <property name="title">
      <string>From File..</string>
     </property>
     <addaction name="actionEncode"/>
     <addaction name="actionDecode"/>
    </widget>
    <addaction name="menuFrom_File"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEncode">
   <property name="text">
    <string>Encode</string>
   </property>
  </action>
  <action name="actionDecode">
   <property name="text">
    <string>Decode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
